availability.php

```diff
@@ -76,6 +76,6 @@
     <form action="availability.php" method="post">
       ' . daily_matrix ( $date, $users ) . '
     </form>
-    ' . print_trailer ( false, true, true );
+    ' . print_trailer ( 0, 1, 1 );
```

Comment:
Well, I left the default parameters as true/false in the functions.
Since we don't send in named parameters, developers have to go look at the function itself to see what's going on anyway.
Once I saw;
function x ($a=false, $b=true) {}
I pretty much knew what incoming 0 and 1 were doing everywhere.

If you really want to keep true/false, I'll stop converting them.